app/playground/page.tsx

```diff
@@ -0,0 +1,10 @@
+import type { FC } from 'react';
+
+// @TODO: implement in this repo
+const PlaygroundPage: FC = () => (
+    <div className="flex justify-center p-4">
+        <iframe src="https://playground.ada-url.com" width="90%" height="500px" />
```

Comment:
I just pushed this before reviewing your code. Can you remove it and update your branch?